runtime/src/asset.rs

```diff
@@ -244,16 +237,16 @@ decl_module! {
             Ok(())
         }
 
-        /// This function is used to transfer a ticker to someone else
+        /// This function is used to accept a ticker transfer
+        /// NB: To reject the transfer, call remove auth function in identity module.
         ///
         /// # Arguments
         /// * `origin` It consist the signing key of the caller (i.e who signed the transaction to execute this function)
-        /// * `to_did` DID of the future owner of the ticker
-        /// * `_ticker` ticker to transfer
-        pub fn approve_ticker_transfer(origin, to_did: IdentityId, _ticker: Vec<u8>) -> Result {
+        /// * `auth_id` Authorization ID of ticker transfer authorization
+        pub fn accept_ticker_transfer(origin, auth_id: u64) -> Result {
```

Comment:
Please just open the file and review this function as a whole. The diff is very confusing. This is essentially a new function that replaces a few of the existing functions.